src/physics.js

```diff
@@ -0,0 +1,81 @@
+import AABB from './aabb';
+import QuadTree from './qtree';
+import Victor from 'victor';
+
+export default class Physics {
+    constructor(levelBoundArr, timeStep = 10, gravity = 9.82) {
+        this.physicsBodyArr = new Array();
+        this.timeStepFactor = timeStep/1000;
+        this.gravity = gravity; //units/(s^2)
+        this.quadTree = new QuadTree(0, 0, levelBoundArr[0], levelBoundArr[1]);
+    }
+
+    addBody(physicsBody) {
+        physicsBody.quadID = this.physicsBodyArr.length;
```

Comment:
The quadrant ID is the same as the current length of this array?  What if multiple boxes are in the same quadrant?